### PR TITLE
[es] Translate Libraries > Distillery section

### PR DESCRIPTION
**Pull request by j-burgos**
This pull request adds a translation to spanish for the Distillery lesson in the Libraries section.

Corrections are really welcome since I'm not sure about the translation of certain concepts or words.

eg. deploy/deployment => desplegar/despliegue
or release task => tarea de release

Thanks in advance!

### CODE REVIEW

File: es/lessons/libraries/distillery.md [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+
+En la terminal, ejecuta:
+
+```
+mix release.init
+```
+
+Este comando genera una carpeta `rel` que contiene algunos archivos de configuración.
+
+Para generar un release ejecuta en la terminal `mix release`
```
Comment (erickgnavar) [inline es/lessons/libraries/distillery.md]: Desde la versión 2 se debe usar `mix distillery.release` para crear el release, al parecer la versión en inglés está desactualizada.

File: es/lessons/libraries/distillery.md [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+- `version` - elimina el cache de la aplicación cuando se actualice en caliente la version de la aplicación.
+- `port` - cambiar el puerto para que sea definido por una variable de ambiente nos permite enviar el número del puerto cuando iniciemos la aplicación. Al iniciar la aplicación podemos pasar el puerto ejecutando `PORT=4001 _build/prod/rel/book_app/bin/book_app foreground`
+
+Si ejecutaste el comando anterior, pudiste haber notado que tu aplicación falló porque no puede conectarse a la base de datos ya que ninguna existe actualmente. Esto puede rectificarse ejecutando un comando `mix` de Ecto. En la terminal, ejecuta lo siguiente:
+
+```
+MIX_ENV=prod mix ecto.create
+```
+
+Este comendo creará la base de datos por ti. Intenta volver a ejecutar tu aplicación y debería iniciar de manera correcta. Sin embargo, puedes notar que las migraciones de la base de datos no se han ejecutado. Por lo general en modo de desarrollo ejecutamos esas migraciones manualmente usando `mix.ecto migrate`. Para el release, tendremos que configurar para que las migraciones se ejecuten automáticamente.
```
Comment (erickgnavar) [inline es/lessons/libraries/distillery.md]: `comendo` -> 'comando`

File: es/lessons/libraries/distillery.md [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+Luego, crea un nuevo archivo - `rel/hooks/post_start/migrate.sh` y agrega el siguiente código:
+
+```
+echo "Running migrations"
+
+bin/book_app rpc "Elixir.BookApp.ReleaseTasks.migrate"
+
+```
+
+Para que ese código se ejecute de manera correcta, utilizaremos el módulo `rpc` de Erlan que nos permite hacer llamadas a procedimientos remotamente. Básicamente, podemos llamar a una función en un nodo remotamente y obtener su respuesta. Cuando estamos en producción es muy probable que nuestra aplicación esté en varios nodos.
```
Comment (erickgnavar) [inline es/lessons/libraries/distillery.md]: `Erlan` -> `Erlang`

**Comment by j-burgos** [comment]
Corrections resolved. I also changed the lesson's title so that it appears correctly translated in the website

**Pull request merged**